lui.el

```diff
@@ -689,7 +689,7 @@ Otherwise, we move to the next button."
       (message "No such symbol %s" name)
       (ding))
      (t
-      (help-xref-interned sym)))))
+      (describe-symbol sym)))))
```

Comment:
That one doesn't exist in 24.5.

Reply:
Than I drop that.
However 24.5 is still very old and I don't see anyone using that still.

Reply:
You can very well support both by using a conditional. The tricky part is to write it in a style that makes the byte-compiler happy, see https://www.gnu.org/software/emacs/manual/html_node/elisp/Compiler-Errors.html and https://www.gnu.org/software/emacs/manual/html_node/elisp/Warning-Tips.html.

Reply:
It seems the only way of doing this is as follows:

```elisp
(with-suppressed-warnings ((obsolete help-xref-interned))
  (if (fboundp 'describe-symbol)
      (describe-symbol sym)
    (help-xref-interned sym)))
```

This is ugly and I doubt worth the change. I've asked emacs-devel whether there's any better alternative.